Samantha-samanthalpark/Homework-4/madlib-with-loops/js/app.js

```diff
@@ -0,0 +1,38 @@
+var startupX = ['Uber', 'Google', 'Amazon', 'Apple', 'Facebook', 'Twitter'];
+var startupY = ['Slack', 'Trello', 'Tesla', 'Hyperloop', 'Harvest'];
+
+const createButton = document.querySelector('#create');
+const favoriteButton = document.querySelector('#save');
+const printButton = document.querySelector('#print');
+
+createButton.addEventListener('click', function(){
+  document.querySelector('h1').innerHTML = '';
+
+  var random1 = Math.floor((Math.random() * startupX.length));
+  var random2 = Math.floor((Math.random() * startupY.length));
+  let madLib = 'A startup that is ' + startupX[random1] + ', but for ' + startupY[random2];
+
+  console.log(madLib);
+
+  const mainHeading = document.createElement('h1');
```

Comment:
This works! Now how could you replace the text of the existing h1, rather than creating a new one each time?